sandstorm.js

```diff
@@ -1,7 +1,9 @@
+import { Meteor } from 'meteor/meteor';
+import { Picker } from 'meteor/communitypackages:picker';
+
 // Sandstorm context is detected using the METEOR_SETTINGS environment variable
 // in the package definition.
-const isSandstorm =
-  Meteor.settings && Meteor.settings.public && Meteor.settings.public.sandstorm;
+const isSandstorm = Meteor.settings?.public?.sandstorm;
```

Comment:
What is this new syntax?

Reply:
It's optional chaining

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining